config/etfuturum/modcompat.cfg

```diff
@@ -0,0 +1,46 @@
+# Configuration file
+
+##########################################################################################################
+# misc
+#--------------------------------------------------------------------------------------------------------#
+# When modded deepslate or raw ore support is enabled, OreDictionary tags will be given to the needed Lord of the Rings ores since they don't have any tags for some reason.
+# LoTR Ores that have no raw/deepslate modded alternative will not be tagged.
+##########################################################################################################
+
+misc {
+    # Disables just the andesite, granite, and diorite full blocks, but leaves their stairs and slabs, etc enabled.
+    # Useful if you have mods like Chisel or Botania which feature these same stones but not the stairs and other variants. [default: false]
+    B:disableBaseBountifulStonesOnly=false
+
+    # Disables copper ingots and ores, but leaves the blocks and other stuff.
+    # Useful if you prefer another mod's copper, but want to use the oxidizing copper building blocks. [default: false]
+    B:disableCopperOreAndIngotOnly=false
```

Comment:
I guess this is the alternative option for copper

Reply:
Done. Set to true so we still have access to all the decorative copper without the EFR ore gen.